docs/outlook/onmessagefromchanged-onappointmentfromchanged-events.md

```diff
@@ -1,41 +1,44 @@
+ms.date: 03/06/2023
 ms.localizationpriority: medium
 ---
 
 # Automatically update your signature when switching between Exchange accounts (preview)
 
-Applying the correct signature to messages when using multiple Exchange accounts is now made easier with the addition of the `OnMessageFromChanged` event to the [event-based activation](autolaunch.md) feature. The `OnMessageFromChanged` event allows your add-in to detect when the account in the **From** field of a message being composed is changed. This event further extends the capabilities of signature add-ins and allows them to:
+Applying the correct signature to messages when using multiple Exchange accounts is now made easier with the addition of the `OnMessageFromChanged` and `OnAppointmentFromChanged` events to the [event-based activation](autolaunch.md) feature. The `OnMessageFromChanged` event occurs when the account in the **From** field of a message being composed is changed, while the `OnAppointmentFromChanged` event occurs when the organizer of a meeting being composed is changed. These events further extend the capabilities of signature add-ins and allow them to:
 
 - Provide users with the convenience to apply custom signatures for each of their accounts.
-- Enable mailbox delegates to more accurately and efficiently manage outgoing messages from multiple mailboxes.
-- Ensure that users' messages meet their organization's communication and marketing policies.
+- Enable mailbox delegates to more accurately and efficiently manage outgoing messages and meeting requests from multiple mailboxes.
+- Ensure that users' messages and appointments meet their organization's communication and marketing policies.
 
 The following sections walk you through how to develop an event-based add-in that handles the `OnMessageFromChanged` event to automatically update a message's signature when the mail account in the **From** field is changed.
 
 > [!IMPORTANT]
 > Features in preview shouldn't be used in production add-ins. We invite you to test this feature in test or development environments and welcome feedback on your experience through GitHub (see the **Feedback** section at the end of this page).
 
 > [!IMPORTANT]
-> The `OnMessageFromChanged` event isn't yet supported for the [Teams manifest for Office Add-ins (preview)](../develop/json-manifest-overview.md).
+> The `OnMessageFromChanged` and `OnAppointmentFromChanged` events aren't yet supported for the [Teams manifest for Office Add-ins (preview)](../develop/json-manifest-overview.md).
 
-## Supported clients and platforms
+## Supported clients
 
-The following table lists client-server combinations that support the `OnMessageFromChanged` event.
+The following table lists the clients that support the `OnMessageFromChanged` and `OnAppointmentFromChanged` events.
```

Comment:
Do you still need to mention that Exchange Online is needed where the feature is supported? Also, Exchange is mentioned in a later section - just confirming that this includes on-premises versions?

Reply:
The events are supported in both Exchange Online and Exchange on-premises environments. Based on our discussion, I'll tackle the supported clients-platforms table in a separate PR. Thanks for brainstorming formats with me, @ElizabethSamuel-MSFT!